CriticalMass/UserDefaults+Additions.swift

```diff
@@ -29,6 +29,17 @@ extension UserDefaults {
         static let usesCounterKey = "usesCounter"
         static let lastRatedVersionKey = "lastRatedVersion"
         static let userNameKey = "username"
+        static let nextRideRadius = "nextRideRadius"
+    }
+
+    /// - Returns: Searchradius in Kilometer. Default value is 20
+    public var nextRideRadius: Int {
+        set { set(newValue, forKey: Keys.nextRideRadius) }
```

Comment:
Maybe I missed it but in which scenarios are we setting a new radius?

Reply:
Atm never bit I thought it would be a nice feature for power users 💪 🦾

Reply:
Makes sense. Maybe we should add a comment to document that